android/maplibreui/src/main/java/com/stadiamaps/ferrostar/maplibreui/views/DynamicallyOrientingNavigationView.kt

```diff
@@ -63,21 +67,27 @@ fun DynamicallyOrientingNavigationView(
         LocationRequestProperties.NavigationDefault(),
     snapUserLocationToRoute: Boolean = true,
     config: VisualNavigationViewConfig = VisualNavigationViewConfig.Default(),
+    currentRoadNameView: @Composable (String?) -> Unit = { roadName ->
+      if (roadName != null) {
+        CurrentRoadNameView(roadName)
+        Spacer(modifier = Modifier.height(8.dp))
+      }
+    },
     onTapExit: (() -> Unit)? = null,
     userContent: @Composable (BoxScope.(Modifier) -> Unit)? = null,
     mapContent: @Composable @MapLibreComposable ((State<NavigationUiState>) -> Unit)? = null,
 ) {
   val orientation = LocalConfiguration.current.orientation
 
-  // Maintain the actual size of the arrival view for MapControl layout purposes.
-  val rememberArrivalViewSize = remember { mutableStateOf(DpSize.Zero) }
-  val arrivalViewSize by rememberArrivalViewSize
+  // Maintain the actual size of the progress view for MapControl layout purposes.
+  val rememberProgressViewSize = remember { mutableStateOf(DpSize.Zero) }
+  val progressViewSize by rememberProgressViewSize
 
   // Get the correct padding based on edge-to-edge status.
   val gridPadding = paddingForGridView()
 
-  // Get the map control positioning based on the arrival view.
-  val mapControls = rememberMapControlsForArrivalViewHeight(arrivalViewSize.height)
+  // Get the map control positioning based on the progress view.
+  val mapControls = rememberMapControlsForProgressViewHeight(progressViewSize.height)
```

Comment:
Looking at the screenshots. I don't like where the annotation and logo are anymore. 

Thinking we may need a custom strategy for ferrostar that doesn't even using the MapLibre functionality. Or we should split them up and put one on the leading and one on the trailing above the progress view, but not the current road view. I'm not _sure_, but I don't think the current street occasionally overlapping these will be a big issue.

Reply:
Will keep thinking about this but current idea is to hide THESE controls while navigating.

Reply:
Opened an issue #318.

For THIS PR we can at least hide the lower controls to match iOS.